doc/source/parameters/unresolved-cfd-dem/cfd-dem.rst

```diff
@@ -41,7 +39,7 @@ This subsection includes parameters related to multiphase flow simulations using
     .. math::
         \Omega = \frac{d_p \omega}{2 \left | u - v \right |}
 
-    where :math:`\omega` is the angular velocity of the particle.
+    where :math:`\omega` is the angular velocity of the particle. We do not recommend using the Magnus lift force.
```

Comment:
The "We do not recommend" sentence could be a warning instead. I would also add a more verbose sentence.